### PR TITLE
feat(swagger): generate client stub for wfm API spec

### PR DESCRIPTION
**Pull request by kmala**
ref https://github.com/deis/workflow-manager-api/issues/80

### CODE REVIEW

**Comment by arschles** [comment]
@kmala is this PR intended to just create the swagger file and generate the client code. If so, I'm fine with that fwiw. Is there an issue for converting this codebase to use the generated client code?

**Comment by kmala** [comment]
@arschles this does nothing but just generate the client code, the swagger specification is part of #43 and the server code is part of https://github.com/deis/workflow-manager-api/pull/135
i want to track all these changes through the original PR which has all the details https://github.com/deis/workflow-manager-api/issues/80

**Comment by arschles** [comment]
@kmala sgtm! I'm ready for a big rebase in https://github.com/deis/workflow-manager/pull/42 when this is merged, and excited to utilize the client code in there too :)

File: Makefile [hunk 1]
```diff
@@ -6,8 +6,10 @@ export GO15VENDOREXPERIMENT=1
 include versioning.mk
 
 DEV_ENV_IMAGE := quay.io/deis/go-dev:0.9.0
+SWAGGER_IMAGE := quay.io/goswagger/swagger:0.5.0
```
Comment (arschles) [inline Makefile]: is there a corresponding repository that contains the sources (`Dockerfile`, etc...) that generated this image?

**Comment by arschles** [comment]
@kmala left one comment at https://github.com/deis/workflow-manager/pull/44#discussion_r63563881. Assuming that's resolved, this LGTM.

Also, you may be able to resolve the tests by adding the following to the `glide.yaml`:

```yaml
package: speter.net/go/exp/math/dec/inf
repo: https://github.com/bellua/inf
vcs: git
```

File: Makefile (hunk 1 above)
Comment (kmala) [inline Makefile]: i think it is https://github.com/go-swagger/go-swagger/blob/master/Dockerfile

Comment (arschles) [inline Makefile]: nevermind, you pointed me to if offline: https://github.com/go-swagger/go-swagger/blob/master/Dockerfile

**Comment by arschles** [comment]
@kmala does this require https://github.com/deis/workflow-manager/pull/43/files before merge?

**Comment by kmala** [comment]
yes...it requires

**Comment by mboersma** [comment]
Can you clean up the commit messages? Something like:
```
feat(swagger): generate client stub for wfm API spec
chore(glide): add swagger to dependencies
feat(swagger): run "generate client" after rebase
```

**Comment by kmala** [comment]
cleaned up the commit messages

**Comment by arschles** [comment]
@kmala does this require https://github.com/deis/workflow-manager-api/pull/138?

**Comment by kmala** [comment]
@arschles no it doesn't

**Comment by arschles** [comment]
@kmala ah, I see. this is only the generated code

**Pull request merged**